wiki/Rules/id.md

```diff
@@ -18,7 +18,7 @@ Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di li
 3. **Jika di tengah pembicaraan terdapat seseorang yang terputus dari jaringan *(timed out)*, di-silence, atau di-restrict, abaikan saja dan jangan mulai membicarakannya.** Jika seseorang di-restrict, kasus mereka akan otomatis jatuh pada ranah yang hanya melibatkan staf administratif osu! dan diri mereka sendiri.
 4. **Selalu jaga norma sportivitas dalam permainan, dan jangan indahkan para pemain yang berbuat curang atau melakukan tindakan-tindakan yang tidak terpuji lainnya.** Sayangnya orang-orang seperti ini akan selalu ada dari waktu ke waktu, dan pada umumnya semakin mereka diperhatikan semakin keonaran mereka akan menjadi-jadi. Laporkan mereka melalui perintah `!report` atau tombol Report pada laman profil mereka dan biarkan semuanya berlalu.
 5. **Jaga topik obrolan pada kanal publik agar tetap sesuai bagi segala usia dan hindari topik NSFW apabila memungkinkan.** Hal-hal yang mengandung unsur-unsur NSFW tidak sejalan dengan arah tujuan komunitas osu! yang terbuka bagi semua usia. Kamu bebas untuk mengirimkan segala hal yang kamu kehendaki dalam pesan-pesan pribadi (selama tidak melampaui batas), namun topik obrolan pada kanal-kanal publik harus selalu dijaga agar dapat dengan nyaman dinikmati oleh semua orang.
```

Comment:
NSFW nya kalau kita ganti ke `yang menyimpang` atau bisa juga `konten dewasa` ?

Reply:
hmm, i don't really agree on this as NSFW isn't simply limited to "adult content" in its matter -- if any it covers anything that is considered to be not safe for public display, including stuffs such as (for example) disturbing footages that aren't by any means "adult" but is still that is not considered to be unsafe for public view